tool/misc.re

```diff
@@ -6,7 +6,6 @@ enum ReplayMenuOp {
 
 fn create_replay_menu(op: ReplayMenuOp) -> Ui{
     let mut recording_name_label = Text { text: "Recording name" };
-
```

Comment:
Even if removing this one line makes sense, it is the only change in this file and doesn't have to do anything with this PR.

Reply:
True.